docs/security/trust/trust_delegation.md

```diff
@@ -0,0 +1,199 @@
+  targets/releases   "" <all paths>  729c7094a8210fd1e780e7b17b7bb55c9a28a48b871b07f65d97baf93898523a   1
+```
+
+You can see the `targets/releases` with its paths and the key ID you just added.
+
+Notary currently has no good way to map collaborators to keys, so we recommend
+that you add and list delegation keys one at a time, and keep a mapping of the key
+IDs to collaborators yourself, should you need to remove a collaborator.
+
+## Removing a delegation from an existing repository
+
+To revoke a collaborator's permission to sign tags for your image, you must know
+the IDs of their keys, because you need to remove their keys from the
+`targets/releases` delegation.
+
+```
+$ notary -d ~/.docker/trust delegation remove docker.io/<username>/<image> targets/releases 729c7094a8210fd1e780e7b17b7bb55c9a28a48b871b07f65d97baf93898523a
+
+Removal of delegation role targets/releases with keys [729c7094a8210fd1e780e7b17b7bb55c9a28a48b871b07f65d97baf93898523a], to repository "docker.io/<username>/<image>" staged for next publish.
+```
+
+The revocation will take effect as soon as you publish:
+
+```
+$ notary -d ~/.docker/trust publish docker.io/<username>/<image>
+```
+
+## Pushing trusted data as a collborator
+
+As a collaborator with a private key that has been added to a repository's
+`targets/releases` delegation, you need to import the private key that you
+generated into Content Trust.
+
+To do so, you can run:
+
+```
+$ notary -d ~/.docker/trust key import delegation.key --role user
+```
+
+where `delegation.key` is the file containing your PEM-encoded private key.
+
+After you have done so, running `docker push` on any repository that
+includes your key in the `targets/releases` delegation will automatically sign
+tags using this imported key.
+
+## `docker push` behavior
+
+When running `docker push` with Docker Content Trust, Docker Engine 1.10 and above
+will attempt to sign and push with the `targets/releases` delegation if it exists.
+If it does not, the targets key will be used to sign the tag, if the key is available.
+
+## `docker pull` behavior
+
+When running `docker pull` with Docker Content Trust on Docker Engine 1.10,
+Docker will attempt to search in the `targets/releases` delegation for the signed
+image tag, and will fall back to the legacy `targets` tags if it does not exist.
+
+This means that if you have an existing image repository with signed tags, and then
+add the `targets/releases` delegation as per the instructions above, the
+original tags signed with the targets keys will still be `docker pull`-able.
+
+Please note that when searching for signed tags, Docker 1.10 may pick
+up on other non-`targets/releases` delegations' signed tags if they exist for
+this image. In Docker 1.11, this behavior is removed such that all `docker pull`
+commands with Docker Content Trust will pull tags only signed by the `targets/releases`
+delegation role or the legacy tags that were signed with the `targets` key.
```

Comment:
maybe be slightly more explicit "legacy tags that were signed _directly_ with the `targets` key"